### PR TITLE
feat: SimpleTable components - styled table elements

### PR DESCRIPTION
**Pull request by Sodik**

### CODE REVIEW

File: packages/ui/src/SimpleTable.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { HTMLAttributes, TdHTMLAttributes, ThHTMLAttributes } from 'react'
+import cls from 'classnames'
+
+import styles from './SimpleTable.module.scss'
+
+export interface SimpleTableProps extends HTMLAttributes<HTMLTableElement> {
+  stripped?: boolean
```
Comment (danad02) [inline packages/ui/src/SimpleTable.tsx:7]:
```suggestion
  striped?: boolean
```

File: packages/ui/__stories__/SimpleTable.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import { SimpleTable } from '../src'
+
+import styles from './SimpleTable.module.scss'
+
+export default {
```
Comment (danad02) [inline packages/ui/__stories__/SimpleTable.stories.tsx:7]: nit: can we also have a link to figma?

**Review by danad02 — APPROVED** [review]
Very cool! Leaving 2 minor comments.

File: packages/ui/__stories__/SimpleTable.stories.tsx (hunk 2 above)
Comment (Sodik) [reply packages/ui/__stories__/SimpleTable.stories.tsx:7]: We don't have this table in figma. This component replaces legacy table. In the future we could update its style @pawelklasa

File: packages/ui/styles/constants/_colors.scss [hunk 3]
```diff
@@ -33,3 +33,5 @@ $colorOverlayBackground: #f2f5f9 !default;
 $colorAccessibilityOutline: #9648bd !default;
 
 $colorShadow: rgba(53, 61, 73, 0.2) !default;
+
+$simpleTableCellBackground: #FAFAFA !default;
```
Comment (fantkolja) [inline packages/ui/styles/constants/_colors.scss:37]: We have this color already $colorNeutralLighter

**Review by fantkolja — CHANGES_REQUESTED** [review]

File: packages/ui/__stories__/SimpleTable.module.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.th {
+  background: #68c200;
```
Comment (utkukaratas) [inline packages/ui/__stories__/SimpleTable.module.scss:2]: hardcoded colors. not available as constants? 🤔

File: packages/ui/src/SimpleTable.module.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  width: 100%;
+  max-width: 100%;
+  background-color: transparent;
+  border-collapse: collapse;
+  border-spacing: 0;
+  color: c.$colorText;
+
+  th,
+  td {
+    padding: 6px 9px;
```
Comment (utkukaratas) [inline packages/ui/src/SimpleTable.module.scss:13]: we use c.$grid (and multiples of it) instead of hardcoded values.

**Review by fantkolja — APPROVED** [review]

File: packages/ui/__stories__/SimpleTable.module.scss (hunk 4 above)
Comment (Sodik) [reply packages/ui/__stories__/SimpleTable.module.scss:2]: This is custom story style. I will rename the file

**Pull request merged**